packages/backend/src/managers/application/applicationManager.ts

```diff
@@ -202,6 +195,15 @@ export class ApplicationManager extends Publisher<ApplicationState[]> implements
       'model-id': model.id,
     });
 
+    // upload model to podman machine if user system is supported
+    if (!recipeComponents.inferenceServer) {
```

Comment:
why does adding the chat format need to change the code inside application manager ?

Reply:
Error on my side